views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -284,6 +299,8 @@ define([
                 this._interruptRecording();
 
                 provider.stop();
+                setState(recorder, recorderStates.STOP);
```

Comment:
Caution, the `setState()` function is also triggering an event using the state as event name.
So this will result in a duplication of `stop` event. (`recorder.trigger('stop')` will be called twice)

Reply:
fixed, changed name